vineyard_offboard:
  ros__parameters:
    home_x: 0.0
    home_y: -5.0
    first_row_x: -10.0
    first_row_y: 0.0
    row_length: 20.0
    row_spacing: 2.5
    num_rows: 10
    speed: 1.0
    lateral_speed: 0.5
    transit_speed: 2.0
    altitude: 2.5
